portal/server/api.js

```diff
@@ -17,18 +18,69 @@ const login = async (username, password) => {
       data: { username, password },
     });
 
+    if (!FEATURE_FLAGS.MAGIC_LINK) {
+      return response.data
+        ? {
+          success: response.data.success,
+          token: response.data.token,
+          user: response.data.user,
+        }
+        : '';
```

Comment:
Oddly this is the existing implementation -- Throwing an error and catching it might be the best way (although as you can see our error handling in this area could be improved)

Reply:
We'll worry about the error handling here in DTFS2-6750